packages/app/cypress/e2e/specs_list_e2e.cy.ts

```diff
@@ -0,0 +1,91 @@
+    cy.get('[data-cy="app-header-bar"]').findByText('Specs').should('be.visible')
+  })
+
+  it('shows the "E2E Specs" label as the header for the Spec Name column', () => {
+    cy.get('[data-cy="specs-testing-type-header"]').should('contain', 'E2E Specs')
+  })
+
+  it('allows you to search and filter the list of specs in the list', () => {
+    cy.get('button').contains('3 Matches')
+
+    cy.get('input').type('content', { force: true })
+
+    cy.get('[data-cy="spec-item"]').should('have.length', 1)
+    .should('contain', 'dom-content.spec.js')
+
+    cy.get('button').contains('1 Match')
+
+    cy.get('input').clear().type('asdf', { force: true })
+
+    cy.get('[data-cy="spec-item"]').should('have.length', 0)
+
+    cy.get('button').contains('0 Matches')
+  })
+
+  // TODO: find a test project that shows git statuses
+  it.skip('shows a git status for each spec', () => {
+
+  })
+
+  it('collapses or expands folders when clicked, hiding or revealing the specs within it', () => {
+    cy.get('[data-cy="spec-item"]').should('contain', 'dom-content.spec.js')
+    cy.get('[data-cy="row-directory-depth-0"]').click()
+    cy.get('[data-cy="spec-item"]').should('not.exist')
+    cy.get('[data-cy="row-directory-depth-0"]').click()
+    cy.get('[data-cy="spec-item"]').should('contain', 'dom-content.spec.js')
+  })
+
+  it('opens the "Create a new spec" modal after clicking the "New Specs" button', () => {
+    cy.get('[data-cy="standard-modal"]').should('not.exist')
+    cy.get('[data-cy="new-spec-button"]').click()
+    cy.get('[data-cy="standard-modal"]').get('h2').contains('Create a new spec')
+    cy.get('button').contains('Scaffold example specs').should('be.visible')
+    cy.get('button').contains('Create new empty spec').should('be.visible')
+    cy.get('button').get('[aria-label="Close"]').click()
+    cy.get('[data-cy="standard-modal"]').should('not.exist')
+  })
+
+  it('has an <a> tag in the Spec File Row that runs the selected spec when clicked', () => {
+    cy.get('[data-selected-spec="true"]').should('not.exist')
+    cy.get('[data-cy="spec-item-link"]').should('have.attr', 'href')
+    cy.get('[data-cy="spec-item-link"]').contains('dom-content.spec.js').click()
+    cy.get('[data-selected-spec="true"]').contains('dom-content.spec.js')
+    cy.get('[data-cy="runnable-header"]').should('be.visible')
+  })
+
+  it('cannot open the Spec File Row link in a new tab with "cmd + click"', async () => {
+    let numTargets
+    let newNumTargets
+
+    await Cypress.automation('remote:debugger:protocol', { command: 'Target.getTargets' }).then((res) => {
+      numTargets = res.targetInfos.length
+    })
+
+    cy.get('[data-cy="spec-item-link"]').click({ metaKey: true }).then(async () => {
+      await Cypress.automation('remote:debugger:protocol', { command: 'Target.getTargets' }).then((res) => {
+        newNumTargets = res.targetInfos.length
+      })
+
+      expect(numTargets).to.eq(newNumTargets)
```

Comment:
have you seen this test case fail? i worry that this can return too soon (before the new tab opens) and always pass no matter what